src/angular/src/app/spark-core-angular/components/sprk-secondary-navigation-item/sprk-secondary-navigation-item.component.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SparkSecondaryNavigationItemComponent } from './sprk-secondary-navigation-item.component';
+import { RouterTestingModule } from '@angular/router/testing';
+
+describe('SparkSecondaryNavigationItemComponent', () => {
+  let component: SparkSecondaryNavigationItemComponent;
+  let fixture: ComponentFixture<SparkSecondaryNavigationItemComponent>;
+  let secondaryNavigationItemElement: HTMLElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [RouterTestingModule],
+      declarations: [
+        SparkSecondaryNavigationItemComponent
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(SparkSecondaryNavigationItemComponent);
+    component = fixture.componentInstance;
+    secondaryNavigationItemElement = fixture.nativeElement.querySelector('li');
+  })
+
+  it('should create itself', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should add classes when additionalClasses has a value', () => {
+    component.additionalClasses = 'sprk-u-man';
+    fixture.detectChanges();
+    expect(secondaryNavigationItemElement.classList.toString()).toEqual('sprk-u-mrm sprk-u-man');
+  })
+
+  it('should set the href to the incoming href value', () => {
+    component.href = "/alerts";
+    fixture.detectChanges();
+    let link = secondaryNavigationItemElement.querySelector('a');
+    expect(link.getAttribute('href')).toEqual('/alerts');
+  });
+
+  it('should set the right class to match the value of spacing (tiny)', () => {
+    component.spacing = 'tiny';
+    fixture.detectChanges();
+    expect(secondaryNavigationItemElement.classList.contains('sprk-u-mrt')).toEqual(true);
```

Comment:
Did you make a PR for scales for the sprk-u-mrt or add it to spark in a different PR. I remember we chatted about it but forgot what you had decided.

Reply:
not yet, will do today or monday